lib/src/screens/category/category_screen.dart

```diff
@@ -113,7 +116,46 @@ class MainScreen extends StatelessWidget {
           ),
         ],
       ),
-      body: const CategoryScreen(),
+      body: (){
+        SchedulerBinding.instance.addPostFrameCallback((timeStamp) async {
+          try {
+            final APIVersion apiVersion = await UserRepository().fetchApiVersion();
+
+            if (!UserRepository().isVersionSupported(apiVersion)) {
+              ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
Here the analyzer is complaining again.
Just add a ignore comment in the line above `// ignore: use_build_context_synchronously`